versions.md

```diff
@@ -96,6 +96,7 @@ Forthcoming
 * Setting vendor_id in the received CacheChange_t for Data and DataFrag.
 * Added new DynamicData to JSON serializer (`json_serialize`).
 * Added new DynamicType to IDL serializer (`idl_serialize`).
+* `eprosima::fastdds::Time_t` moved to `eprosima::fastdds::dds::Time_t`.
```

Comment:
I think this rephrase makes a little bit more sense to give to the user more context about the change
```suggestion
* DDS implementation of `eprosima::fastdds::Time_t` moved to `eprosima::fastdds::dds::Time_t`.
```